WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -128,4 +153,26 @@ class PageParentViewModel
         }
         return children + grandchildren
     }
+
+    internal fun filterListByQuery(list: List<PageItem>, query: String): List<PageItem> {
+        if (query.isEmpty()) {
+            return list
+        } else {
+            val filtered = list.filter { pageItem ->
+                (pageItem.type != PARENT) || // keep the TOP_LEVEL_PARENT and DIVIDER types
+                        (pageItem.type == PARENT &&
+                                (pageItem as ParentPage).title.toLowerCase(Locale.getDefault()).contains(
```

Comment:
`contains` method has an `ignoreCase` parameter which you can use to simplify this expression